### DISCUSSION TITLE
Export alert status metric

### DISCUSSION THREAD

**Discussion by tomershafir** [grafana/grafana#72174] [Alerting]
I think it may be useful that grafana will optionally export alert status metric to be scraped by prometheus, like
`grafana_alert_status` with contextual labels and numeric state representation.
Then, one can have a consolidated view on alerts for a specific use case, for example, SLO alerts for a specific app, using prometheus' flexible QL and grafana's amazing visualization.

It should be configurable with filters.

The assumption I guess is the new alert manager.

(if you think its useful, I can help contributing)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82626. Feel free to continue the discussion around this there. Thank you!
